NextcloudTalk/NCMediaViewerViewController.swift

```diff
@@ -184,6 +212,7 @@ import UIKit
         self.navigationItem.title = mediaPageViewController.navigationItem.title
 
         self.shareButton.isEnabled = (mediaPageViewController.currentImage != nil) || (mediaPageViewController.currentVideoURL != nil)
+        self.showMessageButton.isEnabled = (mediaPageViewController.currentImage != nil) || (mediaPageViewController.currentVideoURL != nil) // TODO include check if msg was loaded
```

Comment:
Since you fetch the context, we don't need to check if the message was loaded (I assume you. mean in the database)

Reply:
This only works after a swipe, not when you open the media view for the first time.